docs/use-cases/pay-per-use-api/index.md

```diff
@@ -0,0 +1,22 @@
+# Pay-per-use API 
+
+> Micro-payments between a service provider and a client.
+
+Let's consider a service of transaction datum resolution in the form of an HTTP API. Given some datum hash, the API can return, if it exists, the pre-image of that hash digest. On the other end, we have a light-wallet provider with full Cardano smart-contract support and extra care for user experience and security. Part of the transaction workflow of the wallet consists of looking up datums and scripts included in transactions it sees, coming from various DApps its users interact with. Doing so, it can better show how a specific transaction will affect a contract and the data it carries. 
```

Comment:
While this is a nice domain-specific example it runs the risk of conflating it with the actual payment channel and it's requirements. No idea how to make that clear and avoid confusion though.

```suggestion
Let's consider a web service for transaction datum resolution in the form of an HTTP API. Given some datum hash, the API can return, if it exists, the pre-image of that hash digest. On the other end, we have a light-wallet provider with full Cardano smart-contract support and extra care for user experience and security. Part of the transaction workflow of the wallet consists of looking up datums and scripts included in transactions it sees, coming from various DApps its users interact with. Doing so, it can better show how a specific transaction will affect a contract and the data it carries. 
```